app/containers/MessageBox/RightButtons.tsx

```diff
@@ -18,7 +19,7 @@ const RightButtons = React.memo(({ showSend, submit, showMessageBoxActions, isAc
 	if (isActionsEnabled) {
 		return <ActionsButton onPress={showMessageBoxActions} />;
 	}
-	return <View style={styles.buttonsWhitespace} />;
+	return !isIOS ? <View style={styles.buttonsWhitespace} /> : null;
```

Comment:
I couldn't see a visual difference in this case here.

Reply:
The `VIEW` was implemented before only for Android, because of this I created this conditional. I was not able to find why this was implemented only for android. But I remember that at some point a long time ago @diegolmello commented with me about this view. So instead create a regression, I implemented this conditional.